spec/controllers/straws_controller_spec.rb

```diff
@@ -2,63 +2,82 @@
 
 RSpec.describe StrawsController, :type => :controller do
   describe "POST /straws" do
-    it "returns a single username from the list of usernames" do
+    it "should return a single username from the list of usernames" do
```

Comment:
I know this is an opinionated issue, but I have an opinion: [don't us should in test titles](http://betterspecs.org/#should).